Application/EdFi.Ods.Features/Notifications/Redis/InitializeRedisNotifications.cs

```diff
@@ -32,6 +32,7 @@ public class InitializeRedisNotifications : IExternalTask
         _notificationsMessageSink = notificationsMessageSink;
     }
 
+    [Obsolete]
```

Comment:
Is this needed to supress a warning? I don't remember it being as part of one of the .NET 8 migration tickets